dipy/reconst/ivim.py

```diff
@@ -0,0 +1,463 @@
+#!/usr/bin/python
```

Comment:
I don't think we need the shebang here, because this never gets run as a module.